debug/testlib.py

```diff
@@ -276,11 +277,11 @@ def __init__(self, address):
 class Gdb(object):
     logfile = tempfile.NamedTemporaryFile(prefix="gdb", suffix=".log")
     logname = logfile.name
+    print "GDB Temporary file: %s" % logname
```

Comment:
I think we should call them temporary, as my understanding is that these get deleted when the program exits, right?